### PR TITLE
charts/victoria-logs-single: add new chart

### PR DESCRIPTION
**Pull request by zekker6**

### CODE REVIEW

File: charts/victoria-logs-single/values.yaml [hunk 1]
```diff
@@ -0,0 +1,265 @@
+# Default values for victoria-logs.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+rbac:
+  # Note: The PSP will only be deployed, if Kubernetes (<1.25) supports the resource.
+  pspEnabled: true
```
Comment (tenmozes) [inline charts/victoria-logs-single/values.yaml:7]: Let's remove that, i think we don't need to add PSP support for the new chart

File: charts/victoria-logs-single/values.yaml [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+  startupProbe: {}
+    #tcpSocket:
+    #  port: http
+    #failureThreshold: 30
+    #periodSeconds: 15
+    #successThreshold: 1
+    #timeoutSeconds: 5
+
+  # -- Security context to be added to server pods
+  securityContext: {}
```
Comment (tenmozes) [inline charts/victoria-logs-single/values.yaml:176]: what do you adding the default security context and specify it as NON-ROOT?

File: hack/helm/victoria-logs-single.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+podDisruptionBudget:
```
Comment (tenmozes) [inline hack/helm/victoria-logs-single.yaml:1]: should we remove helm folder?

**Review by tenmozes — CHANGES_REQUESTED** [review]

File: charts/victoria-logs-single/values.yaml (hunk 2 above)
Comment (zekker6) [reply charts/victoria-logs-single/values.yaml]: Added sane dafaults here.

File: hack/helm/victoria-logs-single.yaml (hunk 3 above)
Comment (zekker6) [reply hack/helm/victoria-logs-single.yaml:1]: No, files in helm folder are used in CI for running testing.  Files in root of `hack/` are used only to run linter.

**Pull request merged**